src/components/Exchange/TradeHistory.js

```diff
@@ -373,16 +374,15 @@ export default function TradeHistory(props) {
           return defaultMsg;
         }
         const longShortDisplay = order.isLong ? t`Long` : t`Short`;
+        const orderTypeText = order.type === INCREASE ? t`Increase` : t`Decrease`;
```

Comment:
Can there be a swap order?

Reply:
This section is only for `Increase` and `Decrease` orders.